ci/build.py

```diff
@@ -69,7 +69,7 @@ def build_docker(platform: str, docker_binary: str) -> None:
     cmd = [docker_binary, "build",
         "-f", get_dockerfile(platform),
         "-t", tag,
-        "."]
+        "docker"]
```

Comment:
isn't -f enough? not sure I understand -f vs PATH from docker run

Reply:
The problem here is that "." requires the paths in the Dockerfiles to be relative to ci/ instead of ci/docker. Maybe I'll update it to be relative to the parent dir of ``get_dockerfile(platform)``.